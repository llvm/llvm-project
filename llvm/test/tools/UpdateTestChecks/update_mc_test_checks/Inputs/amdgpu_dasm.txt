# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -disassemble -show-encoding %s 2>&1 | FileCheck -check-prefixes=CHECK %s

0x00,0x00,0x00,0x7e

0xfd,0xb8,0x0a,0x7f
