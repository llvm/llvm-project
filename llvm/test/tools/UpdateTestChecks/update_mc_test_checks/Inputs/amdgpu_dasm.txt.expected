; NOTE: Assertions have been autogenerated by utils/update_mc_test_checks.py
# RUN: llvm-mc -triple=amdgcn -mcpu=gfx1100 -disassemble -show-encoding %s 2>&1 | FileCheck -check-prefixes=CHECK %s

0x00,0x00,0x00,0x7e
# CHECK: v_nop                                   ; encoding: [0x00,0x00,0x00,0x7e]

0xfd,0xb8,0x0a,0x7f
# CHECK: :[[@LINE-1]]:1: warning: invalid instruction encoding
